Add Hyper-V queries example (#556)
prometheus-community/windows_exporter: docs/collector.hyperv.md

_This collector does not yet have explained examples, we would appreciate your help adding them!_

## Useful queries

> Percent of physical CPU ressources used per VM (on instance "localhost")

Reviewer: Typo
```suggestion
Percent of physical CPU resources used per VM (on instance "localhost")
```

```

> (sum (rate(windows_hyperv_vm_cpu_hypervisor_run_time{instance="localhost"}[1m]))) / ignoring(vm) group_left max (windows_cs_logical_processors{instance="localhost"}) / 100000

Reviewer: This "magic constant" 100000, do we know why it is required? Could it be a bug in the hyperv collector?

Author: It's probably related to the time format. Some king of Hypervisor Tick... I already saw 1000 or 10000 on some places, but 100000 never.
Poor documentation on those from MS :)

Reviewer: There's a factor 10,000,000 that occurs in a couple of similar places for ticks-to-seconds conversions. But let's leave that out of scope for this PR :)

```

> Percent of physical CPU ressources used by all VMs (on all monitored hosts)

Reviewer: Typo
```suggestion
Percent of physical CPU resources used by all VMs (on all monitored hosts)
```

```
(sum by (instance)(rate(windows_hyperv_vm_cpu_total_run_time{}[1m]))) / max by (instance)(windows_cs_logical_processors{}) / 100000
```

> Percent of physical CPU ressources by the Hosts himself (on all monitored hosts)

Reviewer: ```suggestion
Percent of physical CPU resources by the hosts themselves (on all monitored hosts)
```
